app.js

```diff
@@ -1,14 +1,23 @@
 const { getAllCategories } = require('./controllers/categories.controllers');
+const { getReview } = require('./controllers/reviews.controllers');
 const express = require('express');
-
 const app = express();
 
 app.use(express.json());
```

Comment:
delete this - you shouldn't have irrelevant code in your prs. This is used for posts and patchs.